units/crons.go

```diff
@@ -79,6 +79,10 @@ func StartCrons(ctx context.Context, env cedar.Environment, rpcTLS bool) error {
 
 		return queue.Put(ctx, job)
 	})
+	amboy.IntervalQueueOperation(ctx, remote, time.Hour, time.Now(), opts, func(ctx context.Context, queue amboy.Queue) error {
+		job := NewPeriodicChangePointJob()
+		return queue.Put(ctx, job)
+	})
```

Comment:
I would avoid creating seperate interval queue operations when the same queue and interval is used, so I'd put this in the above operation. 

I'd also pass the timestamp in using the `tsFormat` as in line 72

Reply:
I think I've done what you mean? Had a bit of trouble understand though.